BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1897,6 +1936,14 @@ IEnumerator CruiseRoutine()
                             gpe.Current.emit = false;
                         }
                     }
+                //Thrust
+                if (BDArmorySettings.MISSILE_FUEL_DEDUCTION && burnRate > 0 && burnedMass < massToBurn)
+                {
+                    burnedMass = Mathf.Min(burnedMass + burnRate, massToBurn);
+                    ordinanceMass = -burnedMass;
+                    part.UpdateMass();
+                    part.Update();
+                }
 
```

Comment:
Again, here his only needs to be ordinanceMass -= [whatever the cruseBurnRate is(same as Booster...?); the rest is unnecessary.